pyro/autohelp/conf.py

```diff
@@ -14,20 +16,20 @@ def __init__(
 
 
 default_conf = Conf(
```

Comment:
Cheeky change, don't think I'd miss it haha. But given recent changes, this is valid. Keep as is, just wanted to comment.